l10n/Makefile

```
@@ -9,7 +9,7 @@ l10n-push:
 .PHONY: l10n-pull
 l10n-pull:
 	tx pull -a --skip
-	rm -r fr_FR
+	rm -r fr
```

```suggestion
	rm -rf fr_FR
```

We can add the "f" (force) option to the command. Then it will not complain if the directory is already not existing.

`-f, --force           ignore nonexistent files and arguments, never prompt`

That will stop it failing.

If the `fr_FR` translation has been completely removed from Transifex, then we can delete the whole line.